docs/LayoutSpecs.json

```diff
@@ -375,7 +375,8 @@
               "path": "DSCore.DateTime.TimeOfDay"
```

Comment:
`DSCore.DateTime.TimeOfDay` is a leaf node (not in the `none` category), so its iconUrl is obtained from RawTypeData.json:
```
{
      "fullyQualifiedName": "DSCore.DateTime.TimeOfDay",
      "iconUrl": "/src/resources/icons/DSCore.DateTime.TimeOfDay.png",
      "contextData": "DSCore.DateTime.TimeOfDay@var",
      "itemType": "action"
},
```